arctic/chunkstore/chunkstore.py

```diff
@@ -346,7 +346,7 @@ def append(self, symbol, item):
         """
         self.__update(symbol, item, combine_method=self.__concat)
 
-    def update(self, symbol, item, chunk_range=None):
+    def update(self, symbol, item, chunk_range=None, upsert=False, chunk_size=None):
```

Comment:
I can see your point, to me it makes more sense like so, after all you only have to specify a chunksize if you override upsert and set it to True